packages/fx-tunnel-relayer/src/RelayerConfig.ts

```diff
@@ -0,0 +1,25 @@
+import assert from "assert";
+
+export interface ProcessEnv {
+  [key: string]: string | undefined;
+}
+
+export class RelayerConfig {
+  readonly pollingDelay: number;
+  readonly errorRetries: number;
+  readonly errorRetriesTimeout: number;
+  // We reuse the infura key to connect to Polygon and Ethereum networks.
+  readonly infuraApiKey: string;
+  // Any events older than now minus this value (in seconds) will be ignored.
+  readonly lookback: number;
+
+  constructor(env: ProcessEnv) {
+    const { INFURA_API_KEY, POLLING_DELAY, ERROR_RETRIES, ERROR_RETRIES_TIMEOUT, LOOKBACK } = env;
+    assert(INFURA_API_KEY, "INFURA_API_KEY required");
```

Comment:
This is a bit different to how we normally do this. Should we consider a more standard pattern? Perhaps we can pull the key out of the CUSTOM_NODE_URL or some other structure. Otherwise, this bot is a bit different than others requiring some duplicated information presented in the common feed. Wdyt?

Reply:
hm we need to construct providers for two networks and you can't assume that the CUSTOM_NODE_URL is an Infura node. We have two options the way I see it:
- define a POLYGON_CUSTOM_NODE_URL variable in addition to a CUSTOM_NODE_URL
- keep INFURA_API_KEY

Reply:
so if we did `POLYGON_CUSTOM_NODE_URL` then we assume the `CUSTOM_NODE_URL` is for Ethereum L1. that seems _reasnoble_ as the `CUSTOM_NODE_URL` is injected into the bots as part of the common config. wdyt?

Reply:
Ok i'm good with that I'll make the change.